File: api/code-samples/javascript/eth/account/accounts.js
const OpenSdk = require("opensdk-javascript");


( () => {
    const sdk = new OpenSdk(new ApiClient("https://api.baobab.klaytn.net:8651"));
    sdk.eth.accounts({},(err,data,response)=>{
        console.log(data);
    });
}
)()

File: api/code-samples/python/eth/account/accounts.py
import json

from eth.opensdk_python_eth.configuration import Configuration
from eth.opensdk_python_eth.api_client import ApiClient
from eth.opensdk_python_eth.apis.tags.eth_api import Accounts

host = "https://api.baobab.klaytn.net:8651"

eth = Accounts(ApiClient(configuration=Configuration(host=host)))
eth_response = eth.accounts()

print(json.loads(eth_response.response.data))
